src/pages/Pool.vue

```diff
@@ -288,7 +288,19 @@ export default defineComponent({
     });
 
     watch(poolQuery.error, () => {
-      router.push({ name: 'home' });
```

Comment:
@garethfuller how do you feel about showing the alert instead of the redirect?

Reply:
Yeah I'm happy with this. Was initially just a quick fix to handle bad pool ids.